lib/Logging.js

```diff
@@ -65,7 +63,7 @@ class Logging {
     }
 
     Get(name) {
-        if(!this.loggers.has(name)) {
+        if (!this.loggers.has(name)) {
```

Comment:
This likewise looks unrelated.

Reply:
*to the commit

Reply:
This was just styling fixes..although it looks like I don't keep to a style anyway